### ISSUE TITLE
Any chance of “self contained exe” - so app can be used without “net framework installed” requirement?

### ISSUE THREAD

**Issue by slavap** [irusanov/ZenTimings#34]

**Comment by irusanov** [comment]
I don't think that is possible for the current application. I've looked for options in the past, but .NET Native states the code should not use reflection, serialization/deserialization.

**Comment by slavap** [comment]
@irusanov What about this approach: https://learn.microsoft.com/en-us/dotnet/core/deploying/single-file/overview?tabs=cli
It is not native and bundles a lot of net framework together with the app - but will not require to have .net installed.
